packages/core/src/textures/resources/ImageResource.js

```diff
@@ -58,7 +58,7 @@ export default class ImageResource extends BaseImageResource
          * @member {boolean}
          * @default PIXI.settings.CREATE_IMAGE_BITMAP
          */
-        this.createBitmap = options.createBitmap !== false && settings.CREATE_IMAGE_BITMAP && !!window.createImageBitmap;
+        this.createBitmap = (options.createBitmap || settings.CREATE_IMAGE_BITMAP) && !!window.createImageBitmap;
```

Comment:
@ivanpopelyshev does ☝️ this look okay?

Reply:
You are right, we have to patch this logic